mapfishapp/src/main/webapp/app/js/GEOR_cswquerier.js

```diff
@@ -462,6 +485,57 @@ GEOR.cswquerier = (function() {
          * Observable object
          */
         events: observable,
+        
+        /**
+         * APIMethod: init
+         * Initialize this module 
+         *
+         * Parameters:
+         * m - {OpenLayers.Map} The map instance.
+         */
+        init: function(m) { 
+            map = m;
+        },
+
+        /**
+         * APIMethod: getSpatialFilter
+         * Return the search extent to build a csw search filter
+         *
+         * Returns:
+         * {OpenLayers.Filter.Spatial} bbox filter
+         */
+        getSpatialFilter: function() {
+            var f;
+            if (isLimitExtent) {
```

Comment:
I would factorize this contruct, in order to define the filter only once.
Its value should be set with eg:

```js
var v = isLimitExtent ? 
    map.getExtent().clone().transform(
        map.getProjectionObject(), 
        new OpenLayers.Projection("EPSG:4326")
    ) :
    new OpenLayers.Bounds(
         GEOR.config.CSW_FILTER_SPATIAL
    )
```